website/project/views/drafts.py

```diff
@@ -0,0 +1,282 @@
+from flask import request, redirect
+import httplib as http
+from dateutil.parser import parse as parse_date
+import functools
```

Comment:
Small maintainability note: per PEP8, third party libraries should appear below stdlib imports.